var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -65,40 +67,89 @@ class Trilinos(CMakePackage):
-    variant('hdf5',         default=True,  description='Compile with HDF5')
+    variant('hdf5',         default=True,
+            description='Compile with HDF5')
     variant('suite-sparse', default=True,
             description='Compile with SuiteSparse solvers')
     # not everyone has py-numpy activated, keep it disabled by default to avoid
     # configure errors
-    variant('python',       default=False, description='Build python wrappers')
+    variant('python',       default=False,
+            description='Build python wrappers')
     variant('shared',       default=True,
             description='Enables the build of shared libraries')
     variant('debug',        default=False,
             description='Builds a debug version of the libraries')
-    variant('boost',        default=True,  description='Compile with Boost')
-    variant('tpetra',       default=True,  description='Compile with Tpetra')
-    variant('exodus',       default=False, description='Compile with Exodus from SEACAS')
-
-    variant('dtk',          default=False, description='Enable DataTransferKit')
+    variant('boost',        default=True,
+            description='Compile with Boost')
+    variant('tpetra',       default=True,
+            description='Compile with Tpetra')
+    variant('epetra',       default=True,
+            description='Compile with Epetra')
+    variant('exodus',       default=True,
+            description='Compile with Exodus from SEACAS')
+    variant('pnetcdf',      default=False,
+            description='Compile with parallel-netcdf')
+    variant('zlib',         default=False,
+            description='Compile with zlib')
+    variant('stk',          default=False,
+            description='Compile with STK')
+    variant('belos',        default=True,
+            description='Compile with Belos')
+    variant('zoltan',       default=True,
+            description='Compile with Zoltan')
+    variant('zoltan2',      default=True,
+            description='Compile with Zoltan2')
+    variant('amesos',       default=True,
+            description='Compile with Amesos')
+    variant('amesos2',      default=True,
+            description='Compile with Amesos2')
+    variant('ifpack',       default=True,
+            description='Compile with Ifpack')
+    variant('ifpack2',      default=True,
+            description='Compile with Ifpack2')
+    variant('muelu',        default=True,
+            description='Compile with Muelu')
+    variant('fortran',      default=True,
+            description='Compile with Fortran support')
+    variant('ml',           default=True,
+            description='Compile with ML')
+    variant('gtest',        default=True,
+            description='Compile with Gtest')
+    variant('aztec',        default=True,
+            description='Compile with Aztec')
+    variant('x11',          default=False,
+            description='Compile with X11')
+    variant('eti',          default=True,
```

Comment:
why is i called `eti`? probably `instantiate` and `instantiate_cmplx` would be more descriptive.